libraries/aws_eks_cluster.rb

```diff
@@ -43,6 +43,8 @@ def initialize(opts = {})
       @creating              = resp[:status] == 'CREATING'
       @deleting              = resp[:status] == 'DELETING'
       @tags                  = resp[:tags]
+      @enabled_logging_types = resp[:logging][:cluster_logging].select(&:enabled).map { |type| type[:types] }
+      @disabled_logging_types= resp[:logging][:cluster_logging].reject(&:enabled).map { |type| type[:types] }
```

Comment:
Thanks for the PR @voojoe. It seems these fields may not be stubbed out by the AWS API. Updating to this should get us past the build failures:
```
      @enabled_logging_types = resp.dig(:logging, :cluster_logging)&.select(&:enabled)&.map { |type| type[:types] }
      @disabled_logging_types= resp.dig(:logging, :cluster_logging)&.reject(&:enabled)&.map { |type| type[:types] }
```

Reply:
Hi Ross, thanks for the suggestion, I've introduced these changes now.